change "functional" prong to 'primarily consumptive purposes' prong

lex-node opened the pull request:
NOTE: this commit is a 'quick fix,' but there may be deeper issues.--For example, it may be better to talk about whether uses of the _token_ are functional/consumptive (instead of whether 'the Network' is functional/consumptive). 

The Network could be functional/consumptive but the Token might still not have much of a function or consumptive use on the Network. 

Similar points might be made elsewhere re: network vs. token.

lex-node committed:
Update README.md

sambacha commented:
to clarify, you mean consumption similar to that of taking delivery of a commodity at a future date, meaning the underlying transaction purchase is meant for 'usage', correct?

lex-node commented:
> to clarify, you mean consumption similar to that of taking delivery of a commodity at a future date, meaning the underlying transaction purchase is meant for 'usage', correct?

Not really. By "primarily used for consumptive or other non-investment purposes" I mean to invoke the consumer-use doctrine from the line of cases starting with _Housing Foundation, Inc. v. Forman_, 421 U.S. 837 (1975), which held that “when a purchaser is motivated *primarily or exclusively* by a desire to use or consume the item purchased, the securities laws do not apply.” 

I added *primarily or exclusively* as a gloss because the relevant precedents were dealing with cases where both of the following conditions were satisfied: 

1. the transaction was clearly almost exclusively motivated by a consumer purpose *and* 
2. material securities-like profits were almost structurally impossible. 

For example, the context in which the  _Forman_ court held that shares of stock in a condo association were not securities was that: 

- the condos could not be acquired without purchasing the stock;
- the stock could not be transferred except together with the condos;
- there was a fixed ratio of shares to each condominium;
- the shares did not confer separate voting rights (instead, the rule was one vote per resident, no matter how many shares held by a resident); and
- the economic upside of the shares was artificially constrained by perpetual transfer restrictions, including a cap on the future sales price.

_Teague v. Bakker_, 35 F.3d 978 (4th Cir. 1994) had very similar facts (purchasers buying a time-share at a vacation resort), but the mere fact that the time shares were useful was not enough to save them from being deemed securities. Unlike in _Forman_, there was strong evidence that the motivations of the purchasers were _mixed_: i.e., in addition to being motivated by the possibility of using the time-shares, the purchasers were also acquiring the time-shares with a view toward profiting from the pumpamentals inherent to the design of the time-share scheme.

sherlock-shi-x commented:
LGTM, Meson for safe harbour!

paulbarclay commented:
The wording is a little painful to read, but I think this is the right direction to go in - "functional" is likely not strong enough language, as it could allow things that were clear securities to get through a loophole.

sambacha commented:
> > to clarify, you mean consumption similar to that of taking delivery of a commodity at a future date, meaning the underlying transaction purchase is meant for 'usage', correct?
> 
> Not really. By "primarily used for consumptive or other non-investment purposes" I mean to invoke the consumer-use doctrine from the line of cases starting with _Housing Foundation, Inc. v. Forman_, 421 U.S. 837 (1975), which held that “when a purchaser is motivated _primarily or exclusively_ by a desire to use or consume the item purchased, the securities laws do not apply.”
> 
> I added _primarily or exclusively_ as a gloss because the relevant precedents were dealing with cases where both of the following conditions were satisfied:
> 
> 1. the transaction was clearly almost exclusively motivated by a consumer purpose _and_
> 2. material securities-like profits were almost structurally impossible.
> 
> For example, the context in which the _Forman_ court held that shares of stock in a condo association were not securities was that:
> 
> * the condos could not be acquired without purchasing the stock;
> * the stock could not be transferred except together with the condos;
> * there was a fixed ratio of shares to each condominium;
> * the shares did not confer separate voting rights (instead, the rule was one vote per resident, no matter how many shares held by a resident); and
> * the economic upside of the shares was artificially constrained by perpetual transfer restrictions, including a cap on the future sales price.
> 
> _Teague v. Bakker_, 35 F.3d 978 (4th Cir. 1994) had very similar facts (purchasers buying a time-share at a vacation resort), but the mere fact that the time shares were useful was not enough to save them from being deemed securities. Unlike in _Forman_, there was strong evidence that the motivations of the purchasers were _mixed_: i.e., in addition to being motivated by the possibility of using the time-shares, the purchasers were also acquiring the time-shares with a view toward profiting from the pumpamentals inherent to the design of the time-share scheme.


Non-Investment being functionally equivalent to 'non financial commodity' is where I was drawing that conclusion from. More specifically when an agreement, contract, or transaction would fall within the forward contract exclusions from the “swap” and
“future delivery” definitions in the Commodity Exchange Act, ` See Forward Contracts With Embedded Volumetric Optionality`, 79 FR 69073 (Nov. 20, 2014). 

However Forwards with Optionality enable cash-settlement of whatever is unavailable to be physically settled. I see now why you bring up _Forman_, especially the point where, "...the condos could not be acquired without purchasing the stock;", much like consuming a service (a protocol) may not be possible without paying for it ("the protocol's token"). 

Thanks for the clarification

lex-node committed:
uploads "safe harbor X", results of one-day sprint

lex-node committed:
adds a todo

lex-node committed:
misc improvements

improves prelim notes 

adds definition of "extrinsically affiliated"

lex-node committed:
Merge pull request #2 from lex-node/lex-node-patch-2

misc improvements

lex-node committed:
deleted a completed to-do

lex-node committed:
fix some formatting

lex-node committed:
improve section dividers

lex-node committed:
adds autonomous cryptotreasury

defines autonomous cryptotreasury and deletes the corresponding to-do ; adds spaces to remaining to dos

lex-node committed:
tightens "related person" definition & usage

lex-node committed:
fix notice of reliance formatting

lex-node committed:
misc improvements

fix some formatting

delete accidental copypasta of Rule 13d-5

lex-node committed:
misc format fix

lex-node committed:
misc formatting fixes

lex-node committed:
misc formatting fixes

lex-node committed:
adds checkboxes for todos

lex-node committed:
fix typo

lex-node committed:
adds agreement furnishing etc.

* add requirement for initial development team to furnish agreements relating to the autonomous cryptosystem/autonomous cryptotokens
* fix some broken cross-references
* make usage of 'clause' vs. 'section' vs. 'paragraph' consistent

lex-node committed:
adds policy explainer to-do

lex-node committed:
clarifies answer re: excluded systems

clarifies non-exclusive nature of the safe harbor as part of an answer to question about potentially excluded systems/tokens

lex-node committed:
Merge pull request #1 from lex-node/safe-harbor-x-one-day-sprint

uploads "safe harbor X", results of one-day sprint

lex-node committed:
Update README.md

lex-node committed:
adds Q&A re: initDevTeam sales

adds a Q&A and misc other clarifications regarding actual *sales* of tokens by the initial development team

lex-node committed:
adds section dividers for easier reading

lex-node committed:
Update README.md

sydney-abualy committed:
test PR for funsies

sydney-abualy committed:
commit for test PR

sydney-abualy committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Merge pull request #12 from sydney-abualy/clean-typos

Clean typos 1.5.22

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
Update README.md

lex-node committed:
tightens up software terminology

tightens up software terminology and reduces redundancy by defining "source code", "bytecode", "free software", etc., with definitions drawing inspiration from 48 CFR § 1552.239-71 (https://www.law.cornell.edu/cfr/text/48/1552.239-71)

lex-node committed:
clarify relationship to Reg X
